schema/json-schema-draft-04/payment-gateways/gateway-name.json

```diff
@@ -34,6 +34,7 @@
     "Elavon",
     "eMerchantPay",
     "EPG",
+    "E-PRO",
```

Comment:
I noticed you used `EPro` elsewhere. What is the reason for it to be different here?

Reply:
@ptrofimov can you answer this question? I also noticed `ePro` somewhere in tour PRs.

Reply:
Yes, I am gonna change it to EPro.